src/HTMLGenerator/converter.py

```diff
@@ -60,3 +64,8 @@ def convert(top_level_output_directory, top_level_input_directory, json_with_fil
     os.remove(current_configuration_absolute_path)
 
     return
+
+def convertFromCli(jsonFilePath) :
```

Comment:
You could consider renaming this to convert_from_JSON(). Also, [here](python.org/dev/peps/pep-0008/#function-and-variable-names)'s the style guide for Python. I think it's best that we maintain consistency in the code for styles.

Reply:
Changed the name of the fuction to convert_from_JSON().